### DISCUSSION TITLE
Vimsnake.com migrated to pocketbase

### DISCUSSION THREAD

**Discussion by patmood** [pocketbase/pocketbase#742] [Show and tell]
Hey all!

I'm loving this project and the focus on simplicity. To test it out, I migrated my site https://vimsnake.com from firebase to pocketbase.

It's a simple game built using go compiled to wasm. The users score is encrypted client side before being sent to the server. The server is running on fly.io and uses pocketbase as a framework to decrypt the score. This prevents people cheating.

It's a small project, but a good one to test all the features of pocketbase. The code is here https://github.com/patmood/vim_snake for anyone interested.

Looking forward to building more soon and contributing back!

**Comment by benallfree** [comment]
Hello, great job! I added this to [awesome-pocketbase](https://github.com/benallfree/awesome-pocketbase) under the Showcases section.

**Comment by jimafisk** [comment]
Very cool @patmood! Thanks for sharing :)
